java/flight/flight-core/src/main/java/org/apache/arrow/flight/RequestContext.java

```diff
@@ -26,7 +26,7 @@
    * @param key the variable name.
    * @param value the value.
    */
```

Comment:
Missing the @return

Reply:
Removed the return type entirely based on @keeratsingh feedback.